global.json

```diff
@@ -306,7 +306,7 @@
         "githubUrl": "https://github.com/feedhenry/appforms-project-client.git",
         "repoBranch": "refs/heads/master",
         "forcedSelection": true,
-        "category": "Forms",
+        "category": "Cordova",
```

Comment:
Despite mentioning this in the ticket, I would not list this as Cordova. 
This template is created for Drag and Drop apps. It's not meant to be modified by customers (in most of the cases) and specifying this as Cordova may be misleading.